client/driver/docker.go

```diff
@@ -947,3 +962,50 @@ func (h *DockerHandle) run() {
 		}
 	}
 }
+
+func (h *DockerHandle) startMonitoring() {
```

Comment:
The `docker.Stats` is a blocking call, and the call doesn't return until the container exits. This method was created so that we can call a go routine on the `Stats` call, and that method can also return an error so calling `go docker.Stats` would not be possible since we would like to capture the error in case it returns any.